frontend/src/environments/environment.prod.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
   apiUrl: '/api',
   production: true,
-  googleKey: '597145097157-nj02iuobak24qq0pc4olek3rqbk32p20.apps.googleusercontent.com'
+  googleKey: '597145097157-nj02iuobak24qq0pc4olek3rqbk32p20.apps.googleusercontent.com',
+  clientUrl: 'http://localhost:4200',
```

Comment:
If you really need this one

```suggestion
  clientUrl: 'https://recodo.westeurope.cloudapp.azure.com/',
```